pxtblocks/fields/field_gridpicker.ts

```diff
@@ -0,0 +1,321 @@
+/// <reference path="../../localtypings/blockly.d.ts" />
+
+namespace pxtblockly {
```

Comment:
Wanting to keep field editors separate. They shouldn't have to share much of the logic with pxt.blocks. That way we don't have to worry about conflicting util functions etc.